Task4/ChatServer/ClientObject.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Sockets;
+using System.Text;
+
+namespace ChatServer
+{
+    public class ClientObject
+    {
+        public string Id { get; set; }
+
+        public NetworkStream Stream { get; set; }
```

Comment:
Do we really need public 'set' for this property?

Reply:
> Do we really need public 'set' for this property?

Yes.

Reply:
fixed